src/lib/expert_partitioner/dialogs/format.rb

```diff
@@ -61,27 +57,21 @@ def run
       end
     end
 
-
-  private
+    private
```

Comment:
As stated in the Scrum call, I also prefer the alignment you mention. But this is actually the YaST team rubocop configuration (not something agreed by the storage subteam). We should not add a per-repository difference.